extensions/amp-video-iframe/0.1/amp-video-iframe.js

```diff
@@ -181,8 +193,8 @@ class AmpVideoIframe extends AMP.BaseElement {
   /** @override */
   createPlaceholderCallback() {
     const {element} = this;
-    const poster =
-        htmlFor(element)`<amp-img layout=fill placeholder></amp-img>`;
+    const html = htmlFor(element);
```

Comment:
Did this just because I found out that Github and others will color the HTML syntax of string literals tagged with, exactly, `html`.

Reply:
nice